### DISCUSSION TITLE
Default naming scheme directory defaults to GO pkg location

### DISCUSSION THREAD

**Discussion by pnmcosta** [areknoster/hypert#5] [Q&A] [answered]
Hi Arek,

Great work with this, I have a use case where it's really handy to use, but have come across what I believe is an issue, but let me know if you need any more info about my setup, or possibly I'm doing something wrong?

The default naming scheme in my WSL setup with a Fedora distro and go 1.22.3 seems to be always setting the request storage directory to the package's go pkg directory:

```
hypert: using sequential naming scheme in /root/go/pkg/mod/github.com/areknoster/hypert@v0.3.2/testdata/TestDo directory
```

This happens with running `go test ./... -v` directly on the package's directory, but also when running from VSCode and/or the repo's root.

I have worked around it by specifying a naming scheme and setting a relative path for the directory instead, and then it stores it under my package's directory:

```
ns, err := hypert.NewSequentialNamingScheme("./testdata")
if err != nil {
	t.Fatal(err)
}
httpClient := hypert.TestClient(t, true, hypert.WithNamingScheme(ns)) // during the development recordModeOn is set to true
```

Is this the expected behaviour/usage?

Thank you,
Pedro

**Comment by areknoster** [comment]
Hey @pnmcosta, long time no see!
I reproduced the issue locally and will land the fix most likely today. It went unnoticed, because in other places I'm using the lib I've always used custom directory setup. 

Super cool that you're using hypert and I'm happy you managed to make it work with existing API surface 🥇
Best
Arek

**Reply by pnmcosta** [reply]
Hey man, that's awesome really glad I helped. 

It's a great tool and am loving the way it works, I have come across some edge cases that I'll open a discussion later on for you to review.

Keep up the great work 💪

**Comment by areknoster** [comment, marked as answer]
Should be fixed with https://github.com/areknoster/hypert/pull/6
Please update to v0.3.3 and let me know if it works without your workaround
